main.go

```diff
@@ -92,6 +110,10 @@ func main() {
 	}
 }
 
+type bodyData struct {
```

Comment:
You can put this type into the func itself right before it is used. It seems to be the only place where you use it.

Something like:
```go
bodyData := &struct {
	ClusterName string `json:"cluster_name"`
}{}
```